wordpress-mobile/WordPress-Android
# Quick Start Create Site task icon and tap actions

Fixes #15181 

This PR addresses the following:
- Swaps out the "+" icon next to "Create your site" under "Complete" to the gridicon "site" icon.
- Shows a snackbar on tap of the "Customize your site" > "Create your site" item. NOTE: The snackbar must show on the dialog list, not back on the MySite view.

https://user-images.githubusercontent.com/506707/133266199-a39e4a96-d716-4756-a792-9baed3e74727.mp4


To test:
- Launch App
- Log out / Log back in
- On the Login Epilogue view, "Choose a Site" to manage
- Select "Show me Around" on the Quick Start Prompt
- Navigate to My Site tab
- Tap the "Customize your Site" to open the quick task list view
- Tap the Down Caret next to Complete heading
- Tap the "Create your site" row
- Note: A snackbar is shown with the message "Site created! Complete another task.". The task list does not close.

## Regression Notes
1. Potential unintended areas of impact N/A

2. What I did to test those areas of impact (or what existing automated tests I relied on) 
Manual tested

3. What automated tests I added (or what prevented me from doing so) N/A

PR submission checklist:
